### DISCUSSION TITLE
Awesome firmware! Will we see an S1 F4 UBL version?

### DISCUSSION THREAD

**Discussion by Jeebus678** [mriscoc/Ender3V2S1#355] [Ideas]
Been using this firmware on both of my Ender 3 S1 printers, one of which is an F1 and the other is F4. I noticed that there is only a unified bed leveling version released for the F1. Will we see one for the F4? I've been loving the extra features that UBL brings and have always put them to use, and I unfortunately can't say the same about my second printer.

**Comment by mriscoc** [comment]
Currently UBL can work with the Ender-3 S1F4 but the limited program memory of the F4 doesn't allow to enable UBL with the other features, so, each one must to compile his own version disabling the features that do not need.
